README.md

```diff
@@ -116,45 +109,66 @@ Long term, the aim is for the toolchain to also be able to optionally push to a
         |-- odm_orthophoto_log.txt          # Log file
         |-- gdal_translate_log.txt          # Log for georeferencing the png file
 
-##### Viewing your results
-
 Any file ending in .obj or .ply can be opened and viewed in [MeshLab](http://meshlab.sourceforge.net/) or similar software. That includes `pmvs/recon0/models/option-000.ply`, `odm_meshing/odm_mesh.ply`, `odm_texturing/odm_textured_model[_geo].obj`, or `odm_georeferencing/odm_georeferenced_model.ply`. Below is an example textured mesh:
 
-![](https://raw.githubusercontent.com/OpenDroneMap/OpenDroneMap/master/img/tol_text.png)
+![](https://raw.githubusercontent.com/alexhagiopol/OpenDroneMap/feature-better-docker/toledo_dataset_example_mesh.jpg)
 
 You can also view the orthophoto GeoTIFF in QGIS or other mapping software:
 
 ![](https://raw.githubusercontent.com/OpenDroneMap/OpenDroneMap/master/img/bellus_map.png)
 
-#### Using Docker
-
-You can build and run OpenDroneMap in a Docker container:
-
-    export IMAGES=/absolute/path/to/your/project
-    docker build -t opendronemap:latest .
-    docker run -v $IMAGES:/images opendronemap:latest
+## Build and Run Using Docker
 
-Replace /absolute/path/to/your/images with an absolute path to the directory containing your project (where the images are)
-To pass in custom parameters to the `run.py` script, simply pass it as arguments to the `docker run` command.
+(Instructions below apply to Ubuntu 14.04, but the Docker image workflow 
+has equivalent procedures for Mac OS X and Windows. See [docs.docker.com](docs.docker.com))
 
----
+OpenDroneMap is Dockerized, meaning you can use containerization to build and run it without manually performing the
+installation procedure described above and - most importantly - without tampering with the libraries and packages already
+installed on your machine. Docker software is free to install and use in this context. If you don't have it installed,
+see the [Docker Ubuntu installation tutorial] (https://docs.docker.com/engine/installation/linux/ubuntulinux/) and follow the
+instructions up until "Create a Docker group" inclusive. Once Docker is installed, an OpenDroneMap Docker image can be created
+like so:
 
-Here are some other videos, which may be outdated:
+    git clone https://github.com/OpenDroneMap/OpenDroneMap.git
+    cd OpenDroneMap
+    docker build -t packages -f packages.Dockerfile .
+    docker build -t odm_image .
+    docker run -it --user root\
```

Comment:
Agree with adding the user, but can't fix atm. We spent all day trying to get users and groups to work, but we can't because being a non-root user takes away write privileges to the output directories if they are shared with the host machine via "-v" in the run command. No way around being a root user if you want to view the results outside of Docker which is what most people will want. I advise we leave this be.